src/components/MobileMenuBar.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import styled from 'styled-components'
+import logo from '../assets/logo.svg'
+import menu from '../assets/menu.svg'
+import { mediaQueries } from './Common';
```

Comment:
let's try to keep styling consistent here

i think most of the repo uses no semicolons (sorry 310)

Reply:
ohp i think my autolinter did that, mb